File: src/main/java/upbrella/be/store/repository/StoreImageRepository.java
package upbrella.be.store.repository;

import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;
import upbrella.be.store.entity.StoreImage;

public interface StoreImageRepository extends JpaRepository<StoreImage, Long> {

    List<StoreImage> findByStoreDetailId(Long storeDetailId);
}

File: src/main/kotlin/upbrella/be/store/repository/StoreImageRepository.kt
package upbrella.be.store.repository

import org.springframework.data.jpa.repository.JpaRepository
import upbrella.be.store.entity.StoreImage

interface StoreImageRepository : JpaRepository<StoreImage, Long> {
    fun findByStoreDetailId(storeDetailId: Long): List<StoreImage>
}